src/components/layout/footer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
```suggestion
/* eslint-disable no-console */
```
👀  missed out some development config here